.github/workflows/test.yml

```diff
@@ -125,7 +125,7 @@ jobs:
       - name: Start backend
         env:
           EXIT_AFTER_OPENAPI_GENERATION: true
-          ELASTICSEARCH_ENABLED: ${{ matrix.elasticsearch_enabled }}
+          ELASTICSEARCH_ENABLED: yes
```

Comment:
I would use no here,  no need to fire up elastic search just  for api generation

Reply:
I confused this variable a little and thought that without it, the API endpoints will not be generated in the definition as well. I now know this is not correct and will disable it.